omero/sysadmins/publication.rst

```diff
@@ -0,0 +1,96 @@
+#. In order to have both a public webclient and a closed webclient side by
+   side without affecting the existing closed webclient (and thus giving your
+   existing non-public OMERO users an unchanged user experience in their
+   workflows), a dedicated,
+   :doc:`separate web server <unix/install-web/web-deployment>` for servicing
+   the public workflows can be added and configured to point at an existing
+   OMERO.server.
+
+Data configuration
+------------------
+
+#. Once the data is in the dedicated public group, it can be reorganized and
+   renamed to reflect the publication e.g. Projects can be renamed according
+   to the corresponding figure panels in the manuscript while the names of the
+   Datasets could be retained corresponding to different treatment conditions
+   represented in each figure panel. For example, Project
+   `Schleicher_etal_figure7_c <https://omero.lifesci.dundee.ac.uk/webclient/?show=project-27920>`_
+   contains images underlying the publication Figure panel 7c. Some Projects
+   underlie two publication Figure panels, such as Project
+   `Schleicher_etal_figure2_a_c <https://omero.lifesci.dundee.ac.uk/webclient/?show=project-27917>`_
+   where representative images are shown in the first panel and the
+   corresponding quantification is shown in the next panel. This makes clear
+   which original images are underlying which figure panels in the
+   publication.
+#. Data can also be tagged with OMERO tags to enhance the browsing
+   possibilities through these data for any user with basic knowledge of
+   OMERO. For example, see 
+   `<https://omero.lifesci.dundee.ac.uk/webclient/?show=tag-364188>`_). The
+   tags are highlighting the images displayed in the publication figures as
+   images (the other, non-tagged images in the group are the ones used for
+   analysis which produced the published numerical data).
+
+Configuring URLs
+----------------
+
+#. Create a URL pointing to the first Project (corresponding to the first
+   figure in the publication) which can be used for a DOI and data landing
+   page. For example, Project 'Schleicher_etal_figure1_a'
+   `<https://omero.lifesci.dundee.ac.uk/webclient/?show=project-27936>`_
+   corresponds to `<http://dx.doi.org/10.17867/10000109>`_.
+#. Optionally, you can decide on a set pattern of URLs for this and future
+   publications. For example, in Dundee we have established a pattern which
+   supposes every new publication from our institution will be in a separate
+   group, and this group will be directly navigable by the public user using
+   the syntax: “server-address/pub/publication-identifier”. This means for
+   example, `<https://omero.lifesci.dundee.ac.uk/pub/schleicher-et-al-2017>`_
+   is the link where "omero.lifesci.dundee.ac.uk" is the server address, and
+   "schleicher-et-al-2017" is the publication-identifier.
+#. This makes use of redirects (in the NGINX component of the webserver
```

Comment:
Reformulate this point please. First explain that the redirects will be made possible and describe them, then mention the technical component (NGNIX) which facilitates it. Maybe even further, link to some docs of NGNIX which would allow me to simulate this on my server.